mongodb/mongod.init

```diff
@@ -97,7 +99,9 @@ start() {
     [ -x $exec ] || exit 5
     echo -n $"Starting $prog: "
     daemon --pidfile=${pidfile} --user mongodb "$exec --quiet $options run >> $logfile 2>&1 &"
+    [ -n "$starttime" ] && sleep $starttime # Wait some time
```

Comment:
I won't really change mongo init scripts to delay startup (are we sure 10 seconds are the right value?) but would rather look at the chkconfig settings for openuc-saa and make sure process is gracefully shut down.